Assets/MRTK/Core/Providers/InputSimulation/MixedRealityInputSimulationProfile.cs

```diff
@@ -348,7 +348,29 @@ public class MixedRealityInputSimulationProfile : BaseMixedRealityProfile
         /// Apply random offset to the controller position
         /// </summary>
         public float ControllerJitterAmount => controllerJitterAmount;
+
+        [Header("Motion Controller Settings")]
         [SerializeField]
+        [Tooltip("Key to simulate a trigger press (select) on the motion controller")]
+        private KeyBinding motionControllerTriggerKey = KeyBinding.FromKey(KeyCode.H);
```

Comment:
Curious why select is not just the left mouse button?

Reply:
I was just thinking maybe assigning a different keycode rather than the existing one associated with hand (with no specific reason). I am more than happy to change it to the left moue button and please let me know.